testing/environments/latest.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:8.9.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.8.1
```

Comment:
This should be 8.9.0 for the latest release I think, no idea why it ended up as 8.9.2 originally.

Reply:
OK. I'll do a manual change of this.